Dockerfile

```diff
@@ -4,10 +4,8 @@ ENV BOT_ROOT=/opt/errbot
 
 ADD requirements.txt $BOT_ROOT/requirements.txt
 
-RUN apk add --no-cache libffi openssl git \
+RUN apk add --no-cache libffi openssl git gcc libxml2-dev libxslt-dev libc-dev\
```

Comment:
I think its better the previous way (as currently implemented in travis.yml). So no modifications required here.